engine/src/flutter/shell/common/shell.h

```diff
@@ -511,13 +512,13 @@ class Shell final : public PlatformView::Delegate,
   /// any of the threads.
   std::unique_ptr<DisplayManager> display_manager_;
 
-  // protects expected_frame_size_ which is set on platform thread and read on
-  // raster thread
+  // Protects expected_frame_size_ which is set on platform thread and read on
```

Comment:
Looks like this needs to be updated :)

```suggestion
  // Protects expected_frame_constraints_ which is set on platform thread and read on
```